### ISSUE TITLE
Feature: Notify room when user connects or disconnects

### ISSUE THREAD

**Issue by Hendeca** [bryzaguy/transient#3]
I think it would be cool to add a little notification when someone connects or disconnects to the chat, with a timestamp. That way it would be easy to tell if someone else is in the room with you. Also makes it a bit more "chat room" like.

Other possible features:
* List of active users
* Commands to see extra information about a user?

**Comment by bryzaguy** [comment]
Yeah man!  The extra info on users we should split out into a user profile issue.

**Comment by bryzaguy** [comment]
Actually re-reading what you put.  I'm curious what you mean by commands to see extra info.   I was thinking you were meaning like a link from their name to more info about them.  Command sounds like maybe something you type though the more I think about it.
